src/components/Input/SearchInput.module.scss

```diff
@@ -0,0 +1,28 @@
+@import '../../styles/theme';
+
+@mixin path($color, $styles) {
+    svg {
+        path {
+            stroke: $color;
+        }
+    }
+}
+
+.SearchWrapper {
+    border-radius: 1000px !important; // make it pill shaped
```

Comment:
what does `!important` do?

Reply:
It overrides previous set values for `border-radius` - for some reason this `border-radius` is being overridden. Will have to work out why it's getting overridden in another PR, but it's not critical right now.